### ISSUE TITLE
Space in path fails to find .terraform-version

### ISSUE THREAD

**Issue by AustinSmart** [tfutils/tfenv#107]
`tfenv install` fails when the current working directory has a space in the path
```
tfenv install
cat: /Users/<user>/Documents/OneDrive: No such file or directory
```

```
cat .terraform-version
0.11.14
```

I would change the folder name, but OneDrive for macOS has a specific folder structure that includes spaces.

**Comment by Zordrak** [comment]
Can you confirm if this issue is resolved by #111

**Comment by AustinSmart** [comment]
Yes, this issue is resolved. Thanks!
